robot/2023/src/main/java/frc/robot/Robot.java

```diff
@@ -18,8 +19,21 @@
  */
 public class Robot extends TimedRobot {
   private Command m_autonomousCommand;
+  private final Joystick stick = new Joystick(0);
 
+  VictorSPX leftMotorController = new VictorSPX(2);
+  VictorSPX rightMotorController = new VictorSPX(1);
+  VictorSPX screwDriveMotorController = new VictorSPX(3);
   private RobotContainer m_robotContainer;
+  
+  final int LEFT_STICK_VERTICAL = 1;
+  final int RIGHT_STICK_VERTICAL = 5;
+  final int LEFT_TRIGGER = 2;
+  final int RIGHT_TRIGGER = 3;
+
+  int leftMotorControlOutput = 0;
```

Comment:
Are these used anywhere?  If not, might want to remove to avoid confusion. 😵

Reply:
None were used, they've been taken out